spec/requests/user_can_create_an_event_spec.rb

```diff
@@ -48,4 +51,28 @@
       expect(JSON.parse(response.body)['message']).to eq 'Event was NOT created.'
```

Comment:
Why was it not created?

Reply:
We have edited that message to be more specific